### PR TITLE
Fixed a mismatch between Chinese and English documents for the CrossEntropyLoss #50424

### LINKED ISSUE #50424: 中英文文档不匹配和测试案例与版本不匹配的问题
### 文档链接&描述 Document Links & Description

paddle.nn.CrossEntropyLoss的中文文档信息确实的情况。
https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/CrossEntropyLoss_cn.html#crossentropyloss
与
https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/CrossEntropyLoss_en.html#crossentropyloss
还有
https://github.com/PaddlePaddle/Paddle/blob/release/2.4/python/paddle/nn/layer/loss.py#L141

分别是中文的文档，英文文档与代码注释，三方无法匹配，特别是中文和英文的差距极大。
同时还有测试代码里面是结果与当前情况不匹配。
image
image

同时看了该接口的上一级的接口 `cross_entropy`
其中英文文档信息也不完全匹配。

### 请提出你的建议 Please give your suggestion

建议可以进行修改和完善，对于小的问题也可以及时改正。
理解团队的辛苦也修改的工作量大，存在许多历史遗留问题，但是还是希望可以进行改进

### PR DESCRIPTION
**Pull request by InsaneOnion**
### PR types

Bug fixes
### PR changes

Docs
### Description

Fixed a mismatch between Chinese and English documents for the CrossEntropyLoss.
The content of the Chinese and English documents does not match, and the Chinese document only shows one code example but there are actually two. This commit adds the :name identifier to make the Chinese document exactly copy the sample code of the English API document.

cn docs changes see paddlepaddle/docs#6105

fixes #50424

### CODE REVIEW

File: python/paddle/nn/layer/loss.py [hunk 1]
```diff
@@ -302,6 +302,7 @@ class CrossEntropyLoss(Layer):
     Examples:
 
         .. code-block:: python
+          :name: code-example1
```
Comment (SigureMo) [inline python/paddle/nn/layer/loss.py]:
```suggestion
            :name: code-example1
```

File: python/paddle/nn/layer/loss.py [hunk 2]
```diff
@@ -321,6 +322,7 @@ class CrossEntropyLoss(Layer):
             5.33697682)
 
         .. code-block:: python
+          :name: code-example2
```
Comment (SigureMo) [inline python/paddle/nn/layer/loss.py]:
```suggestion
            :name: code-example2
```

File: python/paddle/nn/layer/loss.py [hunk 3]
```diff
@@ -283,7 +283,7 @@ class CrossEntropyLoss(Layer):
 
         - **label** (Tensor)
 
-            1. If soft_label=False, the shape is
+            1. If soft_label=False, the Shape is
```
Comment (SigureMo) [inline python/paddle/nn/layer/loss.py]: 这里为什么要改成大写？

**Review by SigureMo — COMMENTED** [review]
另外麻烦签署下 CLA，中英文 repo 都需要

image

File: python/paddle/nn/layer/loss.py (hunk 3 above)
Comment (InsaneOnion) [reply python/paddle/nn/layer/loss.py]: 抱歉，这里应该是手误vim模式下改错了

**Review by nemonameless — APPROVED** [review]

**Review by sunzhongkai588 — APPROVED** [review]
LGTM for docs

**Review by SigureMo — APPROVED** [review]
LGTMeow 🐾

**Comment by luotao1** [comment]
hi, @InsaneOnion 
- 非常感谢你对飞桨框架的贡献，我们正在运营一个PFCC组织，会通过定期分享技术知识与发布开发者主导任务的形式持续为飞桨框架做贡献，详情可见 https://github.com/luotao1 主页说明。
- 如果你对PFCC有兴趣，请发送邮件至 [ext_paddle_oss@baidu.com](mailto:ext_paddle_oss@baidu.com)，我们会邀请你加入~
